lib/tenant.js

```diff
@@ -1,23 +1,25 @@
 // @flow
 'use strict'
 
-const apis = {
-  ONEBLINK: 'https://auth-api.blinkm.io',
-  CIVICPLUS: 'https://auth-api.transform.civicplus.com',
+const tenants = {
+  ONEBLINK: {
+    url: 'https://auth-api.blinkm.io',
+    iss:
+      'https://cognito-idp.ap-southeast-2.amazonaws.com/ap-southeast-2_o1t3ntGWx',
+  },
+  CIVICPLUS: {
+    url: 'https://auth-api.transform.civicplus.com',
```

Comment:
can we change this property to `apiOrigin` please.

Reply:
yep